# HG changeset patch
# User shelve@localhost
# Date 1556443705 -28800
#      Sun Apr 28 17:28:25 2019 +0800
# Node ID 9aa10e25f21ac2fb77e8f1b68657af74f7188eca
# Parent  09fd754d28f4438dd36e29e8c3c1fdc45d7e088e
changes to: imported patch line2.patch

diff --git a/temp b/temp
--- a/temp
+++ b/temp
@@ -1,3 +1,4 @@
 line1
 line1.1
 line2
+line3
